src/test/java/StringManipulationTest.java

```diff
@@ -20,66 +29,87 @@ public void tearDown() {
     }
 
     @Test
```

Comment:
what if string contains non alphanumeric chars? no test case but maybe that's not even in the scope of this assignment who knows?

Reply:
That edge case was out of the bounds of the assignment. 
Future improvement for other classes for sure